app/dashboards/user_dashboard.rb

```diff
@@ -46,15 +50,14 @@ class UserDashboard < Administrate::BaseDashboard
   # on the model's form (`new` and `edit`) pages.
   FORM_ATTRIBUTES = %i[
     email
+    matches
+    receivers
     confirmed_at
     name
     confirmation_token
   ].freeze
 
-  # Overwrite this method to customize how users are displayed
-  # across all pages of the admin dashboard.
-  #
-  # def display_resource(user)
-  #   "User ##{user.id}"
-  # end
+  def display_resource(user)
+    user.email
```

Comment:
Not that it is wrong, but shouldn't we go with name for this? Any reason for email?

Reply:
Since emails are unique identifiers made more sense than the name which can be duplicated.

Reply:
👍